client/js/views/GameView.vue

```diff
@@ -340,21 +307,24 @@
         @targetedOneOff="beginTargeting($event)"
         @cancel="clearSelection"
       />
+      <game-dialogs
+        :game="game"
```

Comment:
Should we skip defining game as a prop and just read it from the store directly? One the one hand that further couples the component with the store logic, on the other, it's already pulling a bunch of computed props directly from the store, so why not go all-in?